### PR TITLE
Issues 2226

### PR DESCRIPTION
**Pull request by raftaar1191**
## Description
Pr for issues #2226 

## How Has This Been Tested?
See the Video on how I have tested it out
Video Link: http://screencast-o-matic.com/watch/cbXn3021iT

## Screenshots (jpeg or gifs if applicable):
Dashboard Donation Single Page:
image

Dashboard Donor Single Page:
image

Donation Form Single Page Frontend:
image

Donor Profile Page Shortcode:
image

## Types of changes
Bug fix (non-breaking change which fixes an issue)

Related PR:
https://github.com/WordImpress/Give-Gift-Aid/pull/69
https://github.com/WordImpress/Give-Manual-Donations/pull/80
https://github.com/WordImpress/Give-Tributes/pull/169

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

**Comment by raftaar1191** [comment]
@DevinWalker @ravinderk 
Still working on the issues please do not review it now

**Comment by raftaar1191** [comment]
Closing PR till it get completed

**Comment by raftaar1191** [comment]
@DevinWalker @ravinderk 
Please review this Code and Merge

**Comment by DevinWalker** [comment]
@raftaar1191 this is working well but can you fix the merge conflict?

File: assets/js/frontend/give-checkout-global.js [hunk 1]
```diff
@@ -63,8 +63,16 @@ jQuery(function ($) {
 					// Check if user want to show the feilds or not.
 					if (typeof ( response.show_field ) != undefined && true == response.show_field) {
 						$form.find('p#give-card-state-wrap').removeClass('give-hidden');
+
```
Comment (raftaar1191) [inline assets/js/frontend/give-checkout-global.js]: This file was being remove soo add the new code in the below commit 
https://github.com/WordImpress/Give/pull/2284/commits/0e17396b2cc94d60b44e82088f5f3c8391487902

**Comment by raftaar1191** [comment]
@DevinWalker Merge conflict Solved 

Please review and merge the Code

**Pull request merged**